crates/loro-common/src/lib.rs

```diff
@@ -206,37 +220,34 @@ impl ContainerType {
             ContainerType::Text => LoroValue::String(Arc::new(Default::default())),
             ContainerType::Tree => LoroValue::List(Arc::new(Default::default())),
             ContainerType::MovableList => LoroValue::List(Arc::new(Default::default())),
+            #[cfg(feature = "counter")]
+            ContainerType::Counter => LoroValue::I64(0),
+            ContainerType::Unknown(_) => unreachable!(),
         }
     }
 
     pub fn to_u8(self) -> u8 {
         match self {
-            ContainerType::Map => 1,
-            ContainerType::List => 2,
-            ContainerType::Text => 3,
-            ContainerType::Tree => 4,
-            ContainerType::MovableList => 5,
-        }
-    }
-
-    pub fn from_u8(v: u8) -> Self {
-        match v {
-            1 => ContainerType::Map,
-            2 => ContainerType::List,
-            3 => ContainerType::Text,
-            4 => ContainerType::Tree,
-            5 => ContainerType::MovableList,
-            _ => unreachable!(),
+            ContainerType::Map => 0,
```

Comment:
I'm not sure whether 0 has a special meaning in this context. Did all test cases pass?

Reply:
I don't know, but all test cases pass.